js/app.js

```diff
@@ -0,0 +1,50 @@
+(function (){
+  angular
+  .module("wdi_radio", [
+    "ui.router",
+    "ng-token-auth",
+    "songs"
+  ])
+  .config([
+    "$stateProvider",
+    RouterFunction
+  ])
+  .config([
+    "$authProvider",
+    AuthConfigFunction
+  ]);
+
+  function AuthConfigFunction($authprovider) {
+    $authprovider.configure({
+      apiUrl: "http://localhost:3000"
+    });
+  }
+
+  function RouterFunction($stateProvider, $locationProvider){
+    $stateProvider
+    .state("songIndex", {
```

Comment:
How would you look down a route so only authorized user's can access this appropriate actions?